metadata.json

```diff
@@ -7,19 +7,18 @@
   "source": "https://github.com/voxpupuli/puppet-hashi_stack",
   "project_page": "https://github.com/voxpupuli/puppet-hashi_stack",
   "issues_url": "https://github.com/voxpupuli/puppet-hashi_stack/issues",
-  "description": "This module contains shared code for various HashiCorp modules",
   "dependencies": [
     {
       "name": "puppetlabs-stdlib",
-      "version_requirement": ">= 4.13.1 < 7.0.0"
+      "version_requirement": ">= 5.1.0 < 8.0.0"
     },
     {
       "name": "puppetlabs/apt",
-      "version_requirement": ">= 2.0.0 < 8.0.0"
+      "version_requirement": ">= 6.1.0 < 9.0.0"
     },
     {
       "name": "puppet/yum",
-      "version_requirement": ">= 0.9.6 < 5.0.0"
+      "version_requirement": ">= 3.1.1 < 5.0.0"
     },
     {
       "name": "puppetlabs/yumrepo_core",
```

Comment:
I think typically `yumrepo_core` is not listed as a dependency since it's bundled in the regular puppet-agent package.

Reply:
Agreed. I can drop it from here and just make sure it is in the fixtures file instead.

Reply:
dropped